### ISSUE TITLE
Shift banners downwards?

### ISSUE THREAD

**Issue by Pummelfisch** [noatpad/obsidian-banners#36]
Hey i really like the aestetic looks of banners. So thank you first of all for your work.

However i was wondering if there is the posibility to shift the banner downwards? I also use the Breadcrumbs plugin but it interferes with banners and overlaps.

**Comment by noatpad** [comment]
Yeah, there are a few plugins that interfere with things there. There's one ticket specifically for the Breadcrumbs plugin too (#14). I'm thinking about trying to have better compatibility with other plugins down the line, but having a "shift downwards" setting to help mitigate things might be a nice backup solution, so I'll see what I can do

**Comment by Pummelfisch** [comment]
Ah I see. Did not see that issue. Thanks for pointing there. The solutions provided over there help me for now :)
